bin/install_jupyter.sh

```diff
@@ -74,5 +88,18 @@ cp "$BUILD_DIR"/../app-data/jupyter/cartopy_simple.ipynb \
    "$USER_HOME/jupyter/notebooks/projects/CARTOPY/"
 cp -r /home/user/jupyter /etc/skel
 
+# add geoalchemy to easy access to postgis from python
+wget -c --tries=3 --progress=dot:mega \
+    "http://download.osgeo.org/livedvd/data/jupyter/debs/python-geoalchemy2_0.3.0_all.deb"
+dpkg -i
```

Comment:
copy-paste from the script: 

http://download.osgeo.org/livedvd/data/jupyter/debs/python-geoalchemy2_0.3.0_all.deb

copy paste from "http://download.osgeo.org/livedvd/data/jupyter/debs/":

http://download.osgeo.org/livedvd/data/jupyter/debs/python-geoalchemy2_0.3.0_all.deb

the file seems to be there ..